terraform/stacks/dns/stack.tf

```diff
@@ -1036,6 +1036,54 @@ resource "aws_route53_record" "dev2_delegate" {
   ]
 }
 
+resource "aws_route53_record" "cloud_gov_star_fr_cloud_gov_a" {
```

Comment:
these should all have unique values for the second string, shouldn't they? (e.g `cloud_gov_star_fr_cloud_gov_a`)

Reply:
absolutely, my mistake, fix has been pushed.